forge/containers/stub/index.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         this._app = app
 
         const projects = await this._app.db.models.Project.findAll()
-        projects.forEach(project => {
+        projects.forEach(async (project) => {
```

Comment:
Why does this need to be async? it's not doing any work, just pushing to a local variable

Reply:
Looks like a leftover from some changes I refactored. Thanks!
```suggestion
       projects.forEach(project => {
```